content/en/learn.md

```diff
@@ -78,6 +78,6 @@ If you have the basic know-how of NumPy and how to use it, you can try these adv
 
 If NumPy has been significant in your research, and you would like to acknowledge the project in your academic publication, please see [this citation information](/citing-numpy).
 
-***
-
-_If you would like to contribute to this collection of NumPy Learning resources, please submit your recommendation [here via a pull request](https://github.com/numpy/numpy.org/blob/master/content/en/learn.md). Make sure that your submission includes a brief description regarding why it deserves mention on numpy.org and what audience would benefit from it the most._
+## Contribute to this list
+<a name="add-to-this-list"></a>
```

Comment:
Having the anchor to link to is nice. I'm not really wanting to encourage submissions that much though, given that I fully expect to have to reject a large majority of submissions. So a smaller section header would be better I think.

Reply:
Or maybe it doesn't matter. Resolving.